packages/host/app/services/render-service.ts

```diff
@@ -85,6 +85,20 @@ export class CardStoreWithErrors implements CardStore {
   async loaded() {
     await Promise.allSettled(this.#inFlight);
   }
+  getSearchResource<T extends CardDef = CardDef>(
```

Comment:
why was this necessary? is there an interface that this service is fulfilling that required this?